src/Components/CardSection/Index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable arrow-body-style */
```

Comment:
Remove this comment, I believe it is not needed.

Reply:
Done!